README.md

```diff
@@ -75,6 +80,18 @@ Install binaries
 ```
 sudo make install
 ```
+Build Qt plugin for *.heic (Iphone) files
+
+```
+git clone https://github.com/jakar/qt-heif-image-plugin.git
+cd qt-heif-image-plugin
+mkdir -p build
+cd build
+cmake ..
+make
+sudo make install
+```
+
```

Comment:
Is this necessary only to compile from source or as well to just run it? Can I use slide without heic support if I leave this out?